api/src/services/cacheService.js

```diff
@@ -11,7 +11,11 @@ module.exports = {
     console.log('Cache service initialized');
 
     setInterval(async () => {
-      cache.accounts = await accountsService.get();
+      try {
+        cache.accounts = await accountsService.get();
+      } catch (err) {
+        console.log(err);
```

Comment:
Extend the log message with text that can help identify the code. For example, `Failed to get accounts balances. Error: ...`